demos/candlestick-ohlc-pan-zoom-scroller.html

```diff
@@ -163,6 +164,55 @@
 
 				}, false);
 
+
+			
+				/* left axis scale */
+
+				let leftAxisScale = u.root.querySelectorAll(".u-axis")[1];
```

Comment:
great to see use of cutting-edge features :)

this was exactly the use case i had in mind for these rects, too.

Reply:
I'm implementing a trading system, and the graphic visualization in metatrader doesn't support the graphic types I want to create. I will actually use context2d to create cluster footprint charts. vertical scale will be required to implement these features.

as I saw that others wanted this, I decided to implement it in the example to help those who also want these features, I thought your project was really cool and very fast.